CMakeLists.txt

```diff
@@ -0,0 +1,112 @@
+#
+# Public Domain 2014-present MongoDB, Inc.
+# Public Domain 2008-2014 WiredTiger, Inc.
+#  All rights reserved.
+#
+# See the file LICENSE for redistribution information.
+#
+
+cmake_minimum_required(VERSION 3.11.0)
+
+# Try to use CCACHE if available - speeds up build times
```

Comment:
Comments in our code should be complete sentences and end with a period. Would you please review your comments and update as needed?

Reply:
No problem, I've revised the comments and ensured they are complete sentences and end with a period. Hopefully I covered all the cases but let me know if I missed any.